pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                               units='#',
+                               label=('rotation matrix value 00 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat00_cmd + '?',
+                               set_cmd=srotmat00_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat01_cmd = 'qutech:rotmat{}:rotmat01'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat01'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 00 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat01_cmd + '?',
+                               set_cmd=srotmat01_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat10_cmd = 'qutech:rotmat{}:rotmat10'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat10'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 10 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat10_cmd + '?',
+                               set_cmd=srotmat10_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            srotmat11_cmd = 'qutech:rotmat{}:rotmat11'.format(ch_pair)
+            self.add_parameter('ch_pair{}_rotmat_rotmat11'.format(ch_pair),
+                               units='#',
+                               label=('rotation matrix value 11 of' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=srotmat11_cmd + '?',
+                               set_cmd=srotmat11_cmd + ' {}'
+                               # vals=vals.Numbers(1,2048)
+                               )
+            '''
+            TV mode parameters
+            '''
+            sintavg_cmd = 'qutech:tvmode{}:naverages'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_naverages'.format(ch_pair),
+                               units='#',
+                               label=('the number of integration averages' +
+                                      'of ch_pair {} '.format(ch_pair)),
+                               get_cmd=sintavg_cmd + '?',
+                               set_cmd=sintavg_cmd + ' {}',
+                               vals=vals.Numbers(1, 1024)
+                               )
+            stvseg_cmd = 'qutech:tvmode{}:nsegments'.format(ch_pair)
+            self.add_parameter('ch_pair{}_tvmode_nsegments'.format(ch_pair),
+                               units='#',
+                               label=('the number of TV segments of' +
```

Comment:
I propose `Number of segments ch_pair{}`. 

Also, does this mean I can have a per channel setable number of segments? We currently only do this globally (for all channels at once) but I kind of like this extra freedom.